CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
 # sphinx-docs-starter-pack changelog
 
-## Upcoming
+## 1.3.0
 
+* !!!BREAKING: Uses atomic extensions over `canonical-sphinx[full]`. `sphinx-terminal` has changed syntax support
+* Changed `html_baseurl` for better canonical URL support
```

Comment:
Do we have a preference between finite and non-finite verbs in the log?

Reply:
I think consistently using past tense makes the most sense, but I wouldn't make it a blocking issue for the PR.